scripts/build-json/build-page-json.js

```diff
@@ -0,0 +1,71 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+const bcd = require('./resolve-bcd');
+const examples = require('./compose-examples');
+const attributes = require('./compose-attributes');
+const prose = require('./slice-prose');
+const contributors = require('./resolve-contributors');
+
+const writeToFile = (propertyName, json) => {
+  const data = {
+    html: {
+      elements: {
+        [propertyName]: json,
+      }
+    }
+  };
+
+  const dest = path.join(process.cwd(),'packaged/html/elements', `${propertyName}.json`);
```

Comment:
How do you consolidate a URI like `/en-US/docs/Web/HTML/Element/video` to this file? Note the difference between "Element" and "elements". I can appreciate that the slug might be "hTmL" but it feels fragile to assume that the file is called ".../elements/..." based on "Element".

Reply:
Also, regarding my point above, if I already figured out that the file I was looking for was called "packaged/html/elements/video.json" it doesn't make sense to repeat this in the JSON structure. All I need is the "meta". I.e. what's called `json` in the `writeToFile` function.

Reply:
The idea is that you can compile there files into a single structure, then access the JSON for a single item using a query like `html.elements.video`. This again is exactly the way that content is accessed in browser-compat-data.

Reply:
And are we sure that that structure is better? Playing devils advocate here; just because it was done like that before doesn't mean we have to do it here. 

I appreciate that I stumbed on the word "html". Because we've been focussing on https://developer.mozilla.org/en-US/docs/Web/HTML
So once we do other things like JavaScript or CSS or even things like https://developer.mozilla.org/en-US/docs/Learn/Performance this directory will always be "packaged/html/elements". Is that right?

What if there's a name clash? I.e. some web API called the exact same word as a HTML element.

Reply:
> And are we sure that that structure is better? 

Better than the current MDN URL structure? I think so.

>  Playing devils advocate here; just because it was done like that before doesn't mean we have to do it here.

We don't have to, but I'd like to have a good reason for not using the BCD taxonomy.

For one thing, having a common taxonomy is better for both contributors and consumers.

For another thing: what makes this taxonomy difficult is that the number of objects involved is large and the organization of them is diverse. It's easy to design something you think is good, for HTML, and CSS, and HTTP - and then find it breaks down for APIs, or for JS, and have to redo it all.

BCD has gone through this process: it includes >10,000 items covering all of HTML, HTTP, CSS, API, JS, SVG, and more. So it's tested against a lot of the data. It's also used by >7000 pages on MDN and multiple third-party tools.

We can take advantage of all this prior art, or we can decide to do something new. I think unless we see clear problems with BCD we should do the first of these.

> once we do other things like JavaScript or CSS or even things like https://developer.mozilla.org/en-US/docs/Learn/Performance this directory will always be "packaged/html/elements". Is that right?

I'm not sure what this means. CSS properties will be under "packaged/css/properties". CSS at-rules will be under "packaged/css/at-rules".

> What if there's a name clash? I.e. some web API called the exact same word as a HTML element.

APIs are under "api", so it would be at "packaged/api/video".